setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name="django-stomp",
-    version="5.1.1",
+    version="5.1.2",
```

Comment:
This is not a bug fix and it is not a break change either. I imagine the correct one would be 5.2.0. What do you think?

Reply:
done!